horreum-backend/src/test/java/io/hyperfoil/tools/horreum/svc/TestServiceTest.java

```diff
@@ -57,4 +63,58 @@ public void testCreateDelete(TestInfo info) {
       }
    }
 
+   @org.junit.jupiter.api.Test
+   public void testDisableMuteMissingRuns(TestInfo info) throws RollbackException{
+
+      // create test
+      Test test = createExampleTest(getTestName(info));
+      Test response = createTest(test);
+      Watch watch = new Watch();
+      test = checkTestAsExpected(test, response, watch);
+      watch = checkWatchAsExpected(test, watch);
+      watch.mutemissingruns = true;
+      updateWatch(test, watch);
+   }
+
+   private Test checkTestAsExpected(Test test, Test response, Watch watch) {
```

Comment:
The method name suggest validation but you're actually adding a watch...

Reply:
Yes renamed to be more detailed.

Reply:
I still see the old name, `checkTestAsExpected`...

Reply:
Thought I had updated that.